### DISCUSSION TITLE
LTC on H11 Mini

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#1103] [Feature Requests]
I somehow missed the memo that the Mini now supports the 3.5mm Pro Audio Adapter as of a few firmware updates ago:

https://gopro.com/en/us/news/hero11-black-mini-firmware-update-2-10

This got me wondering if LTC sync is now supported on the Mini.

Either way, is it possible to bring RLTC and OLTC to the Mini as part of the next Labs release?

**Comment by dnewman-gpsw** [comment]
If mini give another update, those might port over.  You could try LTC now to see what happens.

**Reply by RikshaDriver** [reply]
Done some testing and LTC is working on H11 Mini... but is a little finnicky.

It doesn't look like it's working without the AUDS=1 hack.

It requires a few seconds to get LTC sync.

Visually, the timecode is displayed intermittently but is fighting for real-estate with AUDS / EXPS / or regular status. Seems like the LCD is confused on what to show and appears to glitch out.

It also appears the 11 Mini reboots after the 3.5mm jack is removed (Jam Sync). I can see the following log message:

`[2024-09-29 23:42:25-delayed ] wake reason:HEARTBEAT_TIMEOUT `
`[2024-09-29 23:42:25 90%   ] Labs:Display Messages `
`[2024-09-29 23:42:29 90%   ] Labs:Main Start `
`[2024-09-29 23:42:29 90%   ] Labs:H22.03.02.50.71 `
`[2024-09-29 23:42:29 90%   ] Labs:Initialize `
`[2024-09-29 23:42:29 90%   ] Continue command:*EXPS=3 `


At least the README can be updated to reflect that the H11 Mini supports LTC. :)

Would be great to get RLTC and OLTC support on the 11 & 11 Mini... and hopefully a fix for the timecode display "glitch"

**Reply by dnewman-gpsw** [reply]
Heartbeat timeout is a crash.  Maybe not working

**Reply by RikshaDriver** [reply]
Timecode is definitely working. I've confirmed with a number of tests by deliberately setting wrong time via QR, then proper time via LTC, but it is a bit finicky... plus glitchy visual feedback makes it a bit harder to confirm sync.

https://github.com/user-attachments/assets/80bae2ac-07ac-4375-bad3-2b6aa9378493
